addons/binding/org.openhab.binding.airvisualnode/ESH-INF/thing/thing-types.xml

```diff
@@ -52,80 +58,46 @@
 		</config-description>
 	</thing-type>
 
-	<!-- Channel group types -->
-
-	<channel-group-type id="Measurements">
-		<label>Measurements</label>
-		<channels>
-			<channel id="co2_ppm" typeId="Co2_ppm" />
-			<channel id="humidity" typeId="Humidity" />
-			<channel id="aqi_cn" typeId="Aqi_cn" />
-			<channel id="aqi_us" typeId="Aqi_us" />
-			<channel id="pm_25" typeId="Pm_25" />
-			<channel id="temp_celsius" typeId="Temp_celsius" />
-			<channel id="temp_fahrenheit" typeId="Temp_fahrenheit" />
-		</channels>
-	</channel-group-type>
-
-	<channel-group-type id="Status">
-		<label>Status</label>
-		<channels>
-			<channel id="battery_level" typeId="system.battery-level" />
-			<channel id="wifi_strength" typeId="system.signal-strength" />
-			<channel id="timestamp" typeId="Timestamp" />
-			<channel id="used_memory" typeId="Used_memory" />
-		</channels>
-	</channel-group-type>
-
 	<!-- Channel types -->
 
-	<channel-type id="Co2_ppm">
-		<item-type>Number</item-type>
+	<channel-type id="Co2">
+		<item-type>Number:Dimensionless</item-type>
 		<label>CO₂ Level</label>
 		<description>CO&#8322; level, ppm</description>
-		<state readOnly="true" pattern="%d ppm" />
+		<category>CarbonDioxide</category>
+		<state readOnly="true" pattern="%d %unit%" />
 	</channel-type>
 
 	<channel-type id="Humidity">
-		<item-type>Number</item-type>
+		<item-type>Number:Dimensionless</item-type>
 		<label>Humidity</label>
 		<description>Humidity, %</description>
-		<state readOnly="true" pattern="%d %%" />
+		<category>Humidity</category>
+		<state readOnly="true" pattern="%d %unit%" />
 	</channel-type>
 
-	<channel-type id="Aqi_cn">
-		<item-type>Number</item-type>
-		<label>AQI (CN)</label>
-		<description>Air Quality Index (Chinese)</description>
-		<state readOnly="true" pattern="%d" />
-	</channel-type>
-
-	<channel-type id="Aqi_us">
-		<item-type>Number</item-type>
-		<label>AQI (US)</label>
+	<channel-type id="Aqi">
+		<item-type>Number:Dimensionless</item-type>
+		<label>AQI</label>
 		<description>Air Quality Index (US)</description>
+		<category>Flow</category>
```

Comment:
I am not sure if this is the right category. What is flowing here? Better leave it out.